But new errors have not nothing to with this PR: 
```openerp.addons.partner_identification.tests.test_partner_identification: ` Dummy CRUD test
2018-04-11 07:14:51,228 22525 ERROR openerp_test openerp.sql_db: bad query: INSERT INTO "res_partner_id_number" ("id", "active", "partner_id", "name", "create_uid", "write_uid", "create_date", "write_date") VALUES(nextval('res_partner_id_number_id_seq'), true, 6, '1234', 1, 1, (now() at time zone 'UTC'), (now() at time zone 'UTC')) RETURNING id
Traceback (most recent call last):
File "/home/travis/odoo-8.0/openerp/sql_db.py", line 247, in execute
res = self._obj.execute(query, params)
IntegrityError: null value in column "category_id" violates not-null constraint
DETAIL:  Failing row contains (4, null, null, null, 2018-04-11 07:14:51.19689, 1234, 1, null, null, 1, 2018-04-11 07:14:51.19689, t, null, null, 6, null).```
```

--- a/partner_contact_birthdate/__openerp__.py
+++ b/partner_contact_birthdate/__openerp__.py
@@ -29,4 +29,5 @@
         "views/res_partner.xml",
     ],
     "post_init_hook": "post_init_hook",
+    "license": "AGPL-3",
 }

--- a/partner_contact_birthdate/hooks.py
+++ b/partner_contact_birthdate/hooks.py
@@ -7,4 +7,4 @@ from openerp.api import Environment
 def post_init_hook(cr, pool):
     env = Environment(cr, SUPERUSER_ID, {})
     env['res.partner'].search(
-        [('birthdate', "!=", False)])._birthdate_inverse()
+        [('birthdate', "!=", False)])._inverse_birthdate()

--- a/partner_contact_birthdate/models.py
+++ b/partner_contact_birthdate/models.py
@@ -32,22 +32,24 @@ class Partner(models.Model):
 
     # Make the old Char field to reflect the new Date field
     birthdate = fields.Char(
-        compute="_birthdate_compute",
-        inverse="_birthdate_inverse",
+        compute="_compute_birthdate",
+        inverse="_inverse_birthdate",
         store=True)
 
-    @api.one
+    @api.multi
     @api.depends("birthdate_date")
-    def _birthdate_compute(self):
+    def _compute_birthdate(self):
         """Store a string of the new date in the old field."""
-        self.birthdate = self.birthdate_date
+        for this in self:
+            this.birthdate = this.birthdate_date
 
-    @api.one
-    def _birthdate_inverse(self):
+    @api.multi
+    def _inverse_birthdate(self):
         """Convert the old Char date to the new Date format."""
-        try:
-            self.birthdate_date = self.birthdate
-        except ValueError:
-            _logger.warn(
-                _("Could not convert '{0.birthdate}' to date in "
-                  "res.partner {0.id} ({0.name}). Skipping.").format(self))
+        for this in self:
+            try:
+                this.birthdate_date = this.birthdate
+            except ValueError:
+                _logger.warn(
+                    _("Could not convert '{0.birthdate}' to date in "
+                      "res.partner {0.id} ({0.name}). Skipping.").format(this))

